mpap/src/crawler.py

```diff
@@ -0,0 +1,154 @@
+import pathlib
+import os
+import sys
+import csv
+from time import sleep
+import shutil
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.select import Select
+from bs4 import BeautifulSoup
+
+
+BASE_URL_MEMBROS_ATIVOS = 'https://www.mpap.mp.br/transparencia/index.php?pg=consulta_folha_membros_ativos'
+BASE_URL_VERBAS_INDENIZATORIAS = 'https://www.mpap.mp.br/transparencia/index.php?pg=consulta_verbas_indenizatorias'
+
+FLAG = ['remuneracao', 'verbas-indenizatorias']
+# Usado para fazer as comparações
+REMUNERACAO = 'remuneracao'
+VERBAS_INDENIZATORIAS = 'verbas-indenizatorias'
+
+
+def crawl(month, year, driver_path, output_path):
+    files = []
+
+    pathlib.Path(output_path).mkdir(exist_ok=True)
+    driver = setup_driver(driver_path, output_path)
+
+    sleep(4)
+    # Roda duas vezes, uma para pegar a planilha de remuneração, e outra para as de verbas indenizatorias
+    for flag in FLAG:
```

Comment:
Ao invés de ser um for, gostaria realizássemos duas chamadas. NOte que não precisaremos mais passar a flag em virtude dos meus comentários abaixo.